src/traces/bar/attributes.js

```diff
@@ -42,7 +42,15 @@ var marker = extendFlat({
         editType: 'style',
         description: 'Sets the opacity of the bars.'
     },
-    pattern: pattern
+    pattern: pattern,
+    cornerradius: {
+        valType: 'any',
+        editType: 'plot',
```

Comment:
```suggestion
        editType: 'calc',
```
because it has an impact on the calc step, I suspect you could generate problems editing `cornerradius` and having it not properly inherited by other traces if we use `editType: 'plot'`.